servers/src/main/java/tachyon/master/file/meta/Inode.java

```diff
@@ -156,10 +167,28 @@ public boolean isFile() {
   }
 
   /**
-   * Reverse the id of the inode. Only used for a delete operation.
+   * Get the pinned flag of the inode
+   *
+   * @return true if the inode is pinned, false otherwise
+   */
+  public synchronized boolean isPinned() {
```

Comment:
Should pinned be an inode property or an inodefile property? How do we handle propagating a directory's pin?